easybuild/easyconfigs/a/Autoconf/Autoconf-2.69-intel-2014b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Autoconf'
+version = '2.69'
+
+homepage = 'http://www.gnu.org/software/autoconf/'
+description = """Autoconf is an extensible package of M4 macros that produce shell scripts to automatically
+ configure software source code packages. These scripts can adapt the packages to many kinds of UNIX-like
+ systems without manual user intervention. Autoconf creates a configuration script for a package from a
+ template file that lists the operating system features that the package can use, in the form of M4 macro calls."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
```

Comment:
makes sense to me, a great deal of them are probably already available

this does imply issues when using `--try-toolchain` though, since it won't update the GCC version in the dep specs

but once https://github.com/hpcugent/easybuild-framework/pull/1306 gets in, we can simply drop the GCC spec for the deps, and the robot should still pick up the GCC easyconfigs for the deps if they're available